atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -232,15 +232,32 @@ def set_pnc_build_metadata(self, extra):
     def set_remote_sources_metadata(self, extra):
         remote_source_result = self.workflow.prebuild_results.get(PLUGIN_RESOLVE_REMOTE_SOURCE)
         if remote_source_result:
-            url = remote_source_result['annotations']['remote_source_url']
+            remote_sources_image_metadata = [
+                {"name": remote_source["name"], "url": remote_source["url"]}
+                for remote_source in remote_source_result
+            ]
+            extra["image"]["remote_sources"] = remote_sources_image_metadata
+
+            remote_sources_typeinfo_metadata = [
+                {
+                    "name": remote_source["name"],
+                    "url": remote_source["url"],
+                    "archives": [
+                        remote_source["remote_source_json"]["filename"],
+                        remote_source["remote_source_tarball"]["filename"],
+                    ],
+                }
+                for remote_source in remote_source_result
+            ]
             remote_source_typeinfo = {
-                KOJI_BTYPE_REMOTE_SOURCES: {
-                    'remote_source_url': url,
-                },
+                KOJI_BTYPE_REMOTE_SOURCES: remote_sources_typeinfo_metadata,
```

Comment:
we should set this only if self.workflow.source.config.remote_sources

and keep that 'remote_source_url' when self.workflow.source.config.remote_source